lib/web/handle.js

```diff
@@ -0,0 +1,53 @@
+const handle = function (req, res) {
+
+        var config = {
+                ip_address: '192.168.13.204',
+                name: "New Samsung Remote"
+        };
+        var Remote = require('new-samsung-remote');
+        var remote = Remote(config);
+        remote.api_active();
+
+        if (req.url == '/') { //check the URL of the current request
+                // set response header
+                res.writeHead(200, { 'Content-Type': 'text/html' });
+
+                // set response content
+                res.write('<html><body><p>waiting for Key for Samsung TV. . .</p></body></html>');
+                res.end();
+        }
+        else if (req.url == "/KEY_MENU") {
```

Comment:
You are hardcoding the action. Do you plan to have a different else branch for each and every action in the list? Get the action from url and use it generally